### PR TITLE
Erik's Pull Request

### PR DESCRIPTION
**Pull request by erikluu**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -36,3 +36,7 @@
     transform: rotate(360deg);
   }
 }
+
```
Comment (bretthickman) [inline src/App.css]: Unused Lines should be removed

File: src/components/LoginForm.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+            id="email"
+            onChange={(e) => setDetails({ ...details, email: e.target.value })}
+            value={details.email}
+          />
+        </div>
+        <div className="form-group">
+          <label html="password">Password:</label>
+          <input
+            type="password"
+            name="password"
```
Comment (bretthickman) [inline src/components/LoginForm.js]: Having type, name, and, id all set to "password" seems off

**Review by bretthickman — COMMENTED** [review]
Basically, everything looks good. Minor visual errors.

File: src/App.js [hunk 3]
```diff
@@ -1,22 +1,39 @@
 import "./App.css";
 import React, { useRef, useEffect, useState } from "react";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import Dashboard from "./components/Dashboard.js";
+import LoginForm from "./components/LoginForm";
```
Comment (crugggiero) [inline src/App.js]: lgtm, nice way to organize

File: src/App.js [hunk 4]
```diff
@@ -1,22 +1,39 @@
...
 
 <link
   rel="stylesheet"
   href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap"
 />;
 
+const testUser = {
+  email: "carlo@dumb.com",
+  password: "imstupid",
+};
```
Comment (crugggiero) [inline src/App.js]: outstanding password

File: src/App.js [hunk 5]
```diff
@@ -1,22 +1,39 @@
...
+      console.log("details do not match");
+      setError("Details do not match.");
+    }
+  };
+
+  const Logout = () => {
+    console.log("Logged out.");
+  };
+
+  return <div>{user.email !== "" ? <Dashboard /> : <LoginForm Login={Login} error={error} />}</div>;
```
Comment (crugggiero) [inline src/App.js]: should change functionality so we can display more than 2 pages

File: src/index.css [hunk 6]
```diff
@@ -1,21 +1,80 @@
...
+  font-weight: 500;
+  margin-bottom: 30px;
+}
+
+form .inner-form .form-group {
+  display: block;
+  width: 300px;
+  margin-bottom: 15px;
+}
+
```
Comment (crugggiero) [inline src/index.css]: lgtm

File: src/components/LoginForm.js [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+            value={details.password}
+          />
+        </div>
+        <input type="submit" value="Login" />
+      </div>
+    </form>
+  );
+}
+
+export default LoginForm;
```
Comment (crugggiero) [inline src/components/LoginForm.js]: lgtm

File: src/App.js (hunk 3 above)
Comment (JackOrt) [reply src/App.js]: Agree about this seeming like a good organization style.

File: src/App.js [hunk 8]
```diff
@@ -1,22 +1,39 @@
...
-    <div>
-      <Map />
-      <SurfSpotCard />
-    </div>
-  );
+  const [user, setUser] = useState({ email: "" });
+  const [error, setError] = useState("");
+  const Login = (details) => {
+    console.log(details);
+    if (details.email === testUser.email && details.password === testUser.password) {
```
Comment (JackOrt) [inline src/App.js]: Might be better to generalize testUser and then hardcode the test user in elsewhere?

File: src/components/LoginForm.js [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+    <form onSubmit={submitHandler}>
+      <div className="form-inner">
+        <h2>Login</h2>
+        {error !== "" ? <div className="error">{error}</div> : ""}
+        <div className="form-group">
+          <label html="email">Email:</label>
+          <input
+            type="text"
+            name="email"
+            id="email"
```
Comment (JackOrt) [inline src/components/LoginForm.js]: Are these supposed to be unique? Fine for now obviously, but eventually will probably need to hash something from the field to generate an ID, or maybe some better method.

**Pull request merged**